package/MDAnalysis/units.py

```diff
@@ -43,6 +43,7 @@
    =========== ============== ===============================================
    length       Å              :math:`10^{-10}` m
    time         ps             :math:`10^{-12}` s
+   mass         u              :math:`1.660538921 \times 10^{-27}` kg
```

Comment:
We should the use the latest value of the mass here

```suggestion
   mass         u              :math:`1.6605390666 \times 10^{-27}` kg
```

Reply:
This is important change but tbh should be reflected in all units otherwise our unit system is inconsistent, see discussions about 2019 changes in #3941. Perhaps this is ok as a temporary fix?? What does @MDAnalysis/coredevs think?

Reply:
I am going to block on this issue.  so people have a day or two to discuss.